docs/index.adoc

```diff
@@ -150,53 +150,38 @@ It *must be unique* for your application.
+`New Zealand or United States or United Kingdom`. You can also use percentage based rollouts and for example, turn your feature "on" only to 50% of the audience.
 
-==== Percentage Splits rules
+When you create a feature and set initial value, it is treated as a default strategy and a default value (set per each environment). If you decide to split your rollout you can add a rollout strategy and set a feature value for this strategy different from the default.
 
-There is a special rule which allows you to target based on percentage as well, which is by default done against
-a single field value provided by the client. By default this is sessionId, if that isn't provided userKey is used, 
-but in all cases you can provide a special field to base your percentage rollouts on.
+You can apply zero or more rollout strategies to a feature. Each rollout strategy can be assigned a different feature value.
 
-Percentage splits can be on their own or they can be applied in conjuction with general targeting rules (i.e. field
-based match rules). More detail of Percentage splits is available below under <<Percentage rollout rules>>.
-
-==== Maximum rules
-
-Each split has zero or more rules associated with it. If it has zero rules, it must at least have a percentage
-associated with it, otherwise it will be ignored. There is no maximum number of rules you can apply.
-
-==== Each Value is an OR condition
+=== Targeting rules
+A rollout strategy consists of one or more rules. That rule can consist of any combination of matching criteria, so we refer to them
+often as targeting rules. 
 
-Each rule is essentially a field, a condition (equals, includes, etc) and one or more values. Whereas each rule
-is an AND condition, each _value_ is an *OR* condition. For exanple, If the `country` is `New Zealand` *OR* `Indonesia` 
-*AND* the custom field `transport`_ is equal to `boat` *OR* `ship`. 
+Each additional rule is always applied as an `AND` condition - the user is using a mobile device that is iOS `AND`
+their country is Australia.
 
-*How many splits can a feature flag have?*
+Each rule is essentially a field, a condition (equals, includes, etc) and one or more values. Whereas each targeting rule
+is an `AND` condition, each _value_ is an `OR` condition. For example, if the `country` is `New Zealand` `OR` `Indonesia`
+`AND` the custom field `payment_method` is equal to `credit_card` *OR* `dircet_debit`.
 
-Imagine you have a feature flag of type string which controls a "submit button color" that can be in multiple states, 
-e.g green, blue, red etc. With rollout strategies, you can serve a `green` value to users on `iOS` devices, 
-`blue` value to users whose emails ending with `gmail.com` and `red` value to users who are in 
-`New Zealand or United States or United Kingdom`.
+Each rollout strategy has zero or more rules associated with it. If it has zero rules and no percentage rule 
```

Comment:
you may wish to delete the "and no percentage rule" - and clarify what kinds of rules there are (3 types).

Reply:
just as a precursor to the next section.